client/src/main/scala/drt/client/components/EgatesScheduleEditor.scala

```diff
@@ -43,6 +43,27 @@ object EgatesScheduleEditor {
             currentState.copy(editing = updatedEditing)
           }
         }
+        val addBank: () => ReactEventFromInput => CallbackTo[Unit] = () => e => {
+          e.persist()
+          scope.modState { currentState =>
+            val maybeUpdatedEditing = currentState.editing.map { editing =>
+              editing.copy(update = editing.update.copy(banks = editing.update.banks :+ EgateBank(IndexedSeq.fill(10)(true))))
```

Comment:
Just question for understanding , why are filling 10 gate ? Is 10 just random number ?

Reply:
Most egate banks are made up of 10 egates. We could add a field to specify the number of egates in a new bank, but I don't think it's really needed for an MVP. I don't think banks are ever more than 10, and for banks less than ten we can just disable the remaining gates